jobserver/models/core.py

```diff
@@ -939,11 +967,14 @@ def get_full_name(self):
         """Support Django's User contract"""
         return self.fullname
 
-    def get_password_reset_url(self):
-        uid = urlsafe_base64_encode(force_bytes(self.pk))
-        token = PasswordResetTokenGenerator().make_token(self)
+    def get_login_url(self, max_age=timedelta(hours=1)):
```

Comment:
15mins seems more appropriate for a login action to me.

Reply:
15 mins might be too short for NHSmail though :disappointed: 

Maybe 30 mins would be a resonable compromise?